src/pages/AddNewCake.js

```diff
@@ -0,0 +1,34 @@
+import "./AddNewCake.css";
+
+export default function AddNewCake() {
+  return (
+    <form className="form">
```

Comment:
Check my comment above about CSS naming convention

Reply:
Will implement it. :)